intrastat_product/models/intrastat_product_declaration.py

```diff
@@ -196,6 +196,17 @@ def _compute_check_validity(self):
         for this in self:
             this.valid = True
 
+    @api.depends("declaration_line_ids.amount_company_currency")
+    def _compute_numbers(self):
+        for this in self:
+            total_amount = 0  # it is an integer
```

Comment:
Why integer? Isn't that value incorrect and you should remove the fields instead? Are they shown in any place?

Reply:
I think this comment is just to inform the field is wrongly declared in line [133 ](https://github.com/OCA/intrastat-extrastat/pull/235/files#diff-74e1197466e3c9a2c08b006d373e86cca69ac389f34c9e4ed38dc24b84018d80R133) as an integer. Is that also possible to change it as a float and approve the merge? My company need to submit the intrastat report in comming days and we're blocked beacause of this issue : )

Reply:
Correct, but I go further, as if this fields are not shown anywhere, better to simply remove them.

Reply:
OK, then better to convert it to float.